src/Annotation/Access.php

```diff
@@ -15,9 +15,14 @@ final class Access implements Annotation
     /**
      * Field access.
      *
-     * @required
-     *
      * @var string
      */
     public $value;
+
+    /**
+     * Should we return null in access denied.
+     *
+     * @var bool
+     */
+    public $nullOnDenied;
```

Comment:
```suggestion
    public bool $nullOnDenied;
```

Reply:
Still not sure about annotation properties type hint as suggested on #729